api/gyresources/endpoints/DiseaseController.py

```diff
@@ -79,6 +86,12 @@ def get(self):
                 for content in result['content']:
                     content.plant = content.plant.__dict__
                     response.append(content)
+                Logger.Logger.create(flask_app.config["ELASTICURL"],
+                                     'Informative',
+                                     'Ok',
+                                     'get()',
+                                     str(response.__dict__),
```

Comment:
response is a list, you cannot convert list to dict. <br />
So, use `str(response)` instead `str(response.__dict__)`

Reply:
fixed